packages/compartment-mapper/test/test-error-handling.js

```diff
@@ -30,6 +30,7 @@ const onError = (t, { error, title }) => {
   } else {
     t.fail();
   }
```

Comment:
Wile the assertion above is redundant now, I kept it as explanation of the purpose of the test. Could be replaced with a long comment block potentially.